scripts/webpack/webpack.dev.babel.js

```diff
@@ -16,13 +16,13 @@ const plugins = [
       styleBundle: '<!-- Style should be in main bundle -->'
     }
   }),
-  new webpack.EnvironmentPlugin([
-    'WEBEX_ACCESS_TOKEN',
-    'WEBEX_CLIENT_ID',
-    'SPACE_ID',
-    'TO_PERSON_EMAIL',
-    'TO_PERSON_ID'
-  ])
+  new webpack.EnvironmentPlugin({
```

Comment:
Why have we made all these entries empty? There was dummy text earlier but now all these are empty strings. Wouldn't that break something?

Reply:
It was giving errors so I had to update the value as suggested in the error. But they will only be empty if process.node.env doesn't return anything